doc/overview.rst

```diff
@@ -684,6 +684,98 @@ For more detailed explanations of the classes that this library provides for aut
 - `API docs for HyperparameterTuner and parameter range classes <https://sagemaker.readthedocs.io/en/stable/tuner.html>`__
 - `API docs for analytics classes <https://sagemaker.readthedocs.io/en/stable/analytics.html>`__
 
+**********************************
+SageMaker Asynchronous Inference
+**********************************
+Amazon SageMaker Asynchronous Inference is a new capability in SageMaker that queues incoming requests and processes them asynchronously.
+This option is ideal for requests with large payload sizes up to 1GB, long processing times, and near real-time latency requirements.
+You can configure Asynchronous Inference scale the instance count to zero when there are no requests to process, thereby saving costs.
+More information about SageMaker Asynchronous Inference can be found in the `AWS documentation <https://docs.aws.amazon.com/sagemaker/latest/dg/async-inference.html>`__.
+
+To deploy asynchronous inference endpoint, you will need to create a ``AsyncInferenceConfig`` object.
+If you create ``AsyncInferenceConfig`` without specifying its arguments, the default ``S3OutputPath`` will
+be ``s3://sagemaker-{REGION}-{ACCOUNTID}/async-endpoint-outputs/{UNIQUE-JOB-NAME}``. (example shown below):
```

Comment:
I think below you have the output being `async-endpoint-output` but here you say `async-endpoint-outputs`. Im fine with either, but I think we should be consistent

Reply:
sry for this. Modifying these two parts concurrently and left some typos. Fixed it now